SmartDeviceLink/SDLScreenshotViewController.h

```diff
@@ -0,0 +1,22 @@
+//
+//  SDLScreenShotViewController.h
+//
+//  Created by Muller, Alexander (A.) on 2/6/17.
+//  Copyright © 2017 Ford Motor Company. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+/**
+ This class interacts with `SDLLockScreenPresenter`. It loads a screenshot of the app window and presents it on itself in an image view. This view controller is then presented on the lock window, which then presents the lock view controller over it.
```

Comment:
I'm not sure I follow what the goal of this class is. Why does a screen capture of the app need to be behind the lock screen?

Reply:
So, what happens is, the lock window has this VC as its root. This VC takes a screenshot of the app's window before the lock window is displayed. Then, when the lock window is displayed, it appears that nothing changed in the UI, when in fact a new window has been displayed over the app window. Then, the lock screen VC modally presents on top of this VC.

Reply:
Got it. Is there any advantage to doing this legwork vs just setting the window background as transparent? When we migrated the lock-screen to a UIWindow base we were able to achieve a similar effect without needing the additional logic. I am not as sure about iOS 7, but at least since iOS 8 this should work.

Reply:
So the `UIWindow` is transparent, but it would still need a `rootViewController` to modally present the lock screen over, right? The rootVC could be transparent as well I suppose.

Reply:
I've attempted to modify the code to work the way you describe, but the backing window or view controller doesn't seem to be transparent. Creating the window this way,

```
_lockWindow = [[UIWindow alloc] initWithFrame:screenFrame];
_lockWindow.backgroundColor = [UIColor clearColor];
_lockWindow.opaque = NO;

UIViewController *rootController = [[UIViewController alloc] init];
rootController.view.backgroundColor = [UIColor clearColor];
rootController.view.opaque = NO;
_lockWindow.rootViewController = rootController;
```